pkg/pod_cleaner/cmd/pod_cleaner.go

```diff
@@ -87,10 +92,13 @@ func main() {
 	// Remove pvc subfolders in parallel.
```

Comment:
`if "pv-"+pipelinerun.ObjectMeta.Name == pvcSubPath.Name()` -> this not usual subpath folder, seems that it's only in `m2-builds.sh`, in real world it's like this -> https://github.com/redhat-appstudio/build-definitions/blob/main/.tekton/pull-request.yaml#L54

Reply:
And appstudio is using subfolders:
```
sh-4.4$ ls /test
devfile-sample-code-with-quarkus  devfile-sample-java-springboot-basic  devfile-sample-python-basic  lost+found
sh-4.4$ ls /test/devfile-sample-code-with-quarkus/                                  
initialbuild-2022-Feb-11_06-56-04  initialbuild-2022-Feb-11_08-48-10
```

Reply:
I changed implementation. PVC cleaner will use pvc subfolders from pipelinerun k8s resource.